cluster/addons.go

```diff
@@ -33,6 +33,8 @@ const (
 
 	IngressAddonJobName            = "rke-ingress-controller-deploy-job"
 	MetricsServerAddonJobName      = "rke-metrics-addon-deploy-job"
+	UserAddonJobName               = "rke-user-addon-deploy-job"
+	UserAddonIncludeJobName        = "rke-user-includes-addons-deploy-job"
```

Comment:
v nit: should this be renamed to `user-addons-include` since the addon field is addons Include

Reply:
I have to double-check this doesn't cause it to not find the job and re-deploy

Reply:
Changing this would not re-deploy, but would not apply the change to remove as it won't find the job by the old name as it will look for the new name. Also, we don't want every cluster to create new jobs for user addons deployment.